docs/variants/asus_kgpe_d16/dasharo-compatibility/flashrom_wp_test_specification.md

```diff
@@ -0,0 +1,203 @@
+# Flashrom Write Protection – Test Specification
+
+## Test cases
+
+### BIOS WP enable test
+
+#### Test description
+
+The test checks if it's possible to set the WP and whether it actually works
+
+#### Test configuration data
+
+1. `FIRMWARE` = coreboot
+
+2. `OPERATING_SYSTEM` = Ubuntu 20.04
+
+#### Test setup
+
+1. Proceed with the [Generic test setup: firmware](https://novacustom.gitlab.io/dasharo-compatibility/dasharo-compatibility/generic-test-setup/#firmware)
+
+2. Proceed with the [Generic test setup: OS installer](https://novacustom.gitlab.io/dasharo-compatibility/dasharo-compatibility/generic-test-setup/#os-installer)
+
+3. Proceed with the [Generic test setup: OS installation](https://novacustom.gitlab.io/dasharo-compatibility/dasharo-compatibility/generic-test-setup/#os-installation)
+
+4. Proceed with the [Generic test setup: OS boot from disk](https://novacustom.gitlab.io/dasharo-compatibility/dasharo-compatibility/generic-test-setup/#os-boot-from-disk)
+
+5. Check if flashrom and cbfstool are installed:  
+`$ dpkg -s apache-perl flashrom`  
+`$ dpkg -s apache-perl cbfstool`  
+
+6. If flashrom and cbfstool aren't installed, proceed with the below commands:
+
+    - In order to build flashrom we will need some packages and libraries.  
+For Debian based distros execute:  
+`sudo apt-get install git make binutils build-essential ca-certificates \`  
+`libpci-dev libftdi-dev libusb-1.0-0-dev`
+
+    - Now clone the flashrom repository and fetch the patchset:  
+`git clone https://github.com/flashrom/flashrom`  
+`cd flashrom`  
+`git fetch https://review.coreboot.org/flashrom refs/changes/13/59713/7 && \`  
+`git checkout FETCH_HEAD`  
+
+    - Build flashrom:  
+`make`
+
+7. Ensure the WP pin jumper on the flash is not placed.
+
+#### Test steps
+
+1. Clear SPI write protection:  
```

Comment:
Maybe, if the test step consists of some commands to run in the terminal, we should describe that fact as follows:

1. Clear SPI write protection by using the following commands:

[twice tab] command

If we do it in that way, we will get the effect as in the rest of the documentation.